packages/compiler-cli/src/ngtsc/typecheck/testing/index.ts

```
@@ -577,10 +577,9 @@ function getFakeMetadataReader(fakeMetadataRegistry: Map<any, DirectiveMeta|null
         null {
           return fakeMetadataRegistry.get(node.debugName) ?? null;
         },
-    getKnown(kind: MetaKind): Iterable<ClassDeclaration> {
+    getKnown(kind: MetaKind): Array<Reference<ClassDeclaration>> {
       switch (kind) {
-        case MetaKind.Directive:
-          return fakeMetadataRegistry.keys();
+        // TODO: This is not needed for these ngtsc tests, but may be wanted in the future.
```

Nit: should we throw instead of returning a false result here?